.github/workflows/ci.yml

```
@@ -8,7 +8,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", '3.11-dev']
```

```suggestion
        python-version: ["3.7", "3.8", "3.9", "3.10", '3.11-dev - 3.11']
```
I think this will make us auto jump to 3.11 when it's out. `"3.11.0-rc - 3.11"` go suggested to me for black's CI